src/biome/text/_model.py

```diff
@@ -53,7 +62,7 @@ def __hash__(self):
         return pickle.dumps(self).__hash__()
 
 
-class PipelineModel(allennlp.models.Model):
+class PipelineModel(pl.LightningModule, Registrable):
```

Comment:
Is there any reason why it can't be extended from `allennlp.Model` as well?

```python 
class PipelineModel(allennlp.models.Model, pl.LightningModule)
```

Looking for as few changes as possible for maintainability as we tackle the transition, I would highly recommend that option, if we still have direct dependencies to allennlp.

Quite of code included in this class could be dropped.

Reply:
This inheritance makes this PR a bit superfluous, if your prefer to tackle this change afterwards, ok with me!
Will close this PR and open the other one for reviewing and discussions.

I will keep this branch locally, to have the changes ready when we want to make the transition to AllenNLP2.0.